deploy/helm/listener-operator/values.yaml

```
@@ -48,7 +48,8 @@ podAnnotations: {}
 labels:
   stackable.tech/vendor: Stackable
 
-podSecurityContext: {}
+podSecurityContext:
+  {}
```

Was this change intentional?
I would prefer the old way for consistency reasons
```suggestion
podSecurityContext: {}
```

---

Not really.. I think it came from yaml-language-server.